client/src/assets/scss/overrides.scss

```diff
@@ -11,9 +11,15 @@
   padding: 0;
 }
 
-.v-application .text-body-1,
+.v-application .text-body-1 {
+  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
+  letter-spacing: normal !important;
+}
+
 .v-application .text-body-2 {
   font-family: 'BCSans', Verdana, Arial, sans-serif !important;
+  letter-spacing: normal !important;
+  line-height: 1.375rem !important; /* 22px */
```

Comment:
Please note these global (from Vuetify) text styles that can be used for various paragraphs in the app. I expect Tracey will call these out as needed.